core/src/main/java/org/apache/calcite/sql/validate/SqlConformance.java

```diff
@@ -102,6 +102,28 @@ public interface SqlConformance {
    */
   boolean isGroupByAlias();
 
+  /**
+   * Value describing how to perform lookup for values defined in a SELECT statement.
```

Comment:
'lookup for values' - these are not values, these are aliases. It's not a SELECT statement, it's the SELECT list.